onnxruntime/core/framework/op_kernel_context_internal.h

```diff
@@ -21,8 +21,9 @@ class OpKernelContextInternal : public OpKernelContext {
                                    IExecutionFrame& frame,
                                    const OpKernel& kernel,
                                    const logging::Logger& logger,
-                                   const bool& terminate_flag)
-      : OpKernelContext(&frame, &kernel, session_state.GetThreadPool(), logger),
+                                   const bool* terminate_flag,
```

Comment:
Why the change to make the terminate_flag optional? The flag ensures an ORT user can terminate a request (e.g. it hits an infinite loop) so making it optional changes what guarantees we can offer. This was important to 1P users.

Reply:
ok fixed.